.github/workflows/build.yml

```diff
@@ -131,17 +148,17 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       # From https://github.com/samuelmeuli/action-electron-builder/issues/42
-      - name: Install libarchive-tools for pacman build # Related https://github.com/electron-userland/electron-builder/issues/4181
+      - name: install libarchive-tools for pacman build # Related https://github.com/electron-userland/electron-builder/issues/4181
         if: startsWith(matrix.os, 'ubuntu')
         run: sudo apt-get install libarchive-tools
 
       # Compile app and upload artifacts
-      - name: Compile & release Electron app
+      - name: build & release electron app
         uses: samuelmeuli/action-electron-builder@v1
         with:
           build_script_name: build
           package_root: app/
-          args: --config electron-builder.config.js
+          args: --config electron-builder.config.js ${{ (github.event_name != 'push' || needs.get_metadata.outputs.branch != 'main') && '--publish never'}}
```

Comment:
Why?

Reply:
I had the impression that it might speed up electron builder but that might be wrong?